src/selection.js

```diff
@@ -124,8 +192,11 @@ function getSelectionCoords() {
           span.appendChild( document.createTextNode("\u200b") );
           range.insertNode(span);
           rect = span.getClientRects()[0];
-          x = rect.left;
-          y = rect.top;
+          if (rect) {
+            x = rect.left;
+            y = rect.top;
+            width = rect.right - rect.left;
```

Comment:
Didn't the code above collapse the selection?  So this width is spurious.  You probably need to do this for both endpoints of the selection, but still the width won't be that accurate as it won't take word wrapping into account... this could be tricky, a TODO is fine if there's no quick solution, just make it clear in the docs on public methods that width/height might be spurious if the browser doesn't support getClientRects natively on ranges.